driver-rabbitmq/deploy/provision-rabbitmq-aws.tf

```diff
@@ -54,6 +55,7 @@ resource "aws_subnet" "benchmark_subnet" {
   vpc_id                  = "${aws_vpc.benchmark_vpc.id}"
   cidr_block              = "10.0.0.0/24"
   map_public_ip_on_launch = true
+  availability_zone       = var.az
```

Comment:
I'm not sure what's best practice, but in other places it uses the format `"${var.az}"`